### PR TITLE
Add minfs POSIX compatible section

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

**Comment by harshavardhana** [comment]
@deekoder can you take a look?

File: README.md [hunk 1]
```diff
@@ -6,22 +6,29 @@ MinFS helps legacy applications use modern object stores with minimal config cha
 
 > Be careful, it is always possible to remove boltdb cache. Cache will be recreated by MinFS synchronizing metadata from the server.
 
-## MinFS RPMs
+## POSIX Compatibility
+> MinFS is not a full POSIX filesystem, neither it intends to be one. MinFS is built simply for applications which like to use modern object stores but are not hardwired to have strict POSIX compatibility. Please choose your use cases wisely.
```
Comment (deekoder) [inline README.md]: MinFS is not a full POSIX filesystem and it does not intend to be one.

Comment (deekoder) [reply README.md]: MinFS is built for applications that needs access to an object store but not require strict POSIX compatibility. Please use MinFS if this fits your needs.

File: README.md [hunk 2]
```diff
@@ -6,22 +6,29 @@ MinFS helps legacy applications use modern object stores with minimal config cha
...
 
-### Minimum Requirements
+Common use cases not suited for MinFS are
```
Comment (deekoder) [inline README.md]: Use cases not suitable for MinFS use are :

File: README.md (hunk 1 above)
Comment (harshavardhana) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (harshavardhana) [reply README.md]: Done

**Pull request merged**